Library/Homebrew/utils/pypi.rb

```diff
@@ -203,7 +203,7 @@ def update_python_resources!(formula, version: nil, package_name: nil, extra_pac
     end
 
     formula.resources.each do |resource|
-      if !print_only && !resource.url.start_with?(PYTHONHOSTED_URL_PREFIX)
+      if !(ignore_non_pypi_packages || print_only) && !resource.url.start_with?(PYTHONHOSTED_URL_PREFIX)
```

Comment:
```suggestion
      if !ignore_non_pypi_packages && !print_only && !resource.url.start_with?(PYTHONHOSTED_URL_PREFIX)
```
This is a bit more readable, no?